cdk/visit/lambda_code/register_user/register_user.py

```diff
@@ -85,27 +87,24 @@ def add_user_info(self, user_info):
             grad_sem = user_info['GradSemester']
             grad_year = user_info['GradYear']
 
-        majorList = []
-        for major in user_info['Major']:
-            majorList.append({'S': major})
 
-        minorList = []
-        for minor in user_info.get('Minor', []):
-            minorList.append({'S': minor})
+        marshal_item = lambda s: {"S": s}
```

Comment:
Can you add a brief comment about what `marshall_item` means? Future devs likely won’t know what marshaling is.

Reply:
Rewrote this part. Would have looked much nicer if I could chain the map (like in js/ts), but I feel like the functional method in python was more cluttered. Now im using a list comprehension instead, which I think is much cleaner and easier to read. Also added a comment alluding to the attribute type format.